VATSpy.dat

```diff
@@ -18038,8 +18039,9 @@ LPPO-TMA|Santa Maria Radar|LPPO|LPPO-TMA
 LPAM|Lisboa Military||LPPC
 LQSB|Sarajevo|LQSA|LQSB
 LRBB|Bucharest||LRBB
-LSAS|Swiss|LSAG|LSAS
-LSAS|Swiss|LSAZ|LSAS
+LSAS|Swiss||LSAS
+LSAS|Swiss|LSAG|LSAG
+LSAS|Swiss|LSAZ|LSAZ

```

Comment:
Since you defined new FIR's for these two you would want to use these new FIR defintions? 

LSAG|Swiss||LSAG
LSAZ|Swiss||LSAZ

Also a description which would diffrenciate them from LSAS would be nice here, such as a sector name for example. You can see this being done in for example in France and Germany. Not nessecerily a must, but it would be more described.

Reply:
Yes that would be correct as you write, i am Checking at the Moment if they are both Named Swiss or if we Change to Zurich and Geneva. and then i will correct.